packages/uswds-core/src/styles/functions/color/luminance.scss

```diff
@@ -17,10 +17,26 @@
 
 @function luminance($color) {
   $lum: (
-      list.nth($luminance-color-component-values, red($color) + 1) * 0.2126
+      list.nth(
+          $luminance-color-component-values,
+          color.channel($color, "red", $space: rgb) + 1
+        ) *
```

Comment:
**Question:** Is there a reason some of the `$space` values are `rgb` and others are `hsl`?

I'm curious if we should pick one for consistency or if the space is determined by something else

Reply:
In my own broader understanding - I believe hsl uses more "precise" color values for things like hue, saturation, lightness etc.

However - these are not interchangeable. These are the errors that run when they're fipped:
```
 Exception during run: Error: $channel: Color hsl(161.3333333333, 32.3741007194%, 27.2549019608%) has no channel named red.
   ╷
22 │           color.channel($color, "red", $space: hsl) + 1
```
```
 Exception during run: Error: $channel: Color #1b1b1b has no channel named lightness.
   ╷
17 │     color.channel(color($token-from-bg), "lightness", $space: rgb) > 50%,
```

Reply:
Oh I see! That makes sense, RGB have red, green, blue channels and HSL is hue, saturation, lightness.

Looks like each test is matching the required channel so lgtm! Thanks for the explanation Mandy!